pyseir/inference/infer_rt.py

```diff
@@ -578,26 +589,9 @@ def get_posteriors(self, timeseries_type, plot=False):
             plt.close()
         start_idx = -len(posteriors.columns)
 
-        return dates[start_idx:], times[start_idx:], posteriors
-
-    def infer_all(self, plot=True, shift_deaths=0):
-        """
-        Infer R_t from all available data sources.
+        return dates[start_idx:], times[start_idx:], posteriors, start_idx
 
-        Parameters
-        ----------
-        plot: bool
-            If True, generate a plot of the inference.
-        shift_deaths: int
-            Shift the death time series by this amount with respect to cases
-            (when plotting only, does not shift the returned result).
-
-        Returns
-        -------
-        inference_results: pd.DataFrame
-            Columns containing MAP estimates and confidence intervals.
-        """
-        df_all = None
+    def get_available_timeseries(self):
```

Comment:
add docstring for what it does now that it is abstracted from the main infer_all call.